Test/UnitTest1.cs

```diff
@@ -0,0 +1,42 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// UnitTest1.cs
+// Test case for Expression Evaluator.
+// ------------------------------------------------------------------------------------------------
+using Eval;
+namespace TestTraining;
+
+[TestClass]
+public class TestTraining {
+   [TestMethod]
+   public void TestParse () {
+      Evaluator eval = new ();
+      // Expressions with their expected results
+      Dictionary<string, double> exp = new () {
+       { "-10 ^ 2", 100 }, { "a = 4", 4 }, { "b = 3.5", 3.5 }, { "a + b", 7.5 }, { "asin sin 90", 90 },

```

Comment:
-10 ^ 2 = -100 right?

Reply:
No Senior, 
it is like this(-10)^2=100.

Reply:
Okay